tests/integration/examples/test_notebooks_python.py

```diff
@@ -236,6 +236,7 @@ def test_cornac_bpr_integration(
 
 
 @pytest.mark.integration
+@pytest.mark.experimental
```

Comment:
Haven't been following some of the latest code changes, but do we have this new `pytest.marker` defined anywhere in our configuration?

Reply:
right now there is no pipeline for the experimental, as we improve the dependency installation, we will put back some of the tests in the normal pipeline (cpu, gpu or spark). Also, see https://github.com/microsoft/recommenders/pull/1606#issuecomment-1011237669